sklearn/linear_model/tests/test_base.py

```diff
@@ -598,7 +606,8 @@ def test_preprocess_copy_data_no_checks(is_sparse, to_copy):
         assert np.may_share_memory(X_, X)
 
 
-def test_dtype_preprocess_data():
+def test_dtype_preprocess_data(global_random_seed):
+    rng = check_random_state(global_random_seed)
```

Comment:
updated for consistency: `check_random_state` is used in some functions, but not all

Reply:
```suggestion
    rng = np.random.RandomState(global_random_seed)
```